listenbrainz_spark/stats/user.py

```diff
@@ -0,0 +1,193 @@
+import sys
+import listenbrainz_spark
+import time
+from datetime import datetime  
+from listenbrainz_spark.stats_writer.stats_writer import StatsWriter
+from listenbrainz_spark import config
+
+LIMIT = 100
+month = datetime.now().month
```

Comment:
Why are these two here? They are only used in main, so they should be set there.